from cpp_header_tools.utils.exceptions import GeneratedException


class GenerateDriver:
    pass
